.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @mobilecoinfoundation/coredev
```

Comment:
This is fine for now, but we should update it ASAP.

Reply:
moved over to #4